core/src/Revolution/Processors/Security/Group/GetNodes.php

```diff
@@ -147,15 +150,21 @@ public function prepareGroup(modUserGroup $group)
         $c->where(['parent' => $group->get('id')]);
         $c->limit(1);
         $count = $this->modx->getCount(modUserGroup::class, $c);
-        return [
+        $itemArray = [
             'text' => htmlentities($group->get('name'), ENT_QUOTES, 'UTF-8') . ' (' . $group->get('id') . ')',
             'id' => 'n_ug_' . $group->get('id'),
-            'leaf' => $count <= 0,
+            'hasChildren' => $count > 0,
             'type' => 'usergroup',
             'qtip' => $group->get('description'),
             'cls' => $cls,
+            'allowDrop' => true,
             'iconCls' => 'icon icon-group',
         ];
-    }
 
+        if (!$itemArray['hasChildren']) {
+            $itemArray['expanded'] = true;
```

Comment:
What's the reason to leave out the `expanded` array key when `false`?

Reply:
By default nodes are collapsed, no reason to explicitly set that.